tests/rptest/wasm/wasm_build_tool.py

```diff
@@ -77,15 +77,29 @@ def _build_source(self, artifact_dir):
 
     def _run_npm_cmd(self, artifact_dir, cmd):
         with DirectoryContext(artifact_dir) as _:
-            output = subprocess.run(['npm'] + cmd,
-                                    capture_output=True,
-                                    text=True,
-                                    check=False)
-            if output.returncode != 0:
-                raise WasmBuildToolException(
-                    f"Encountered error building wasm script: {output.stderr}")
-            self._redpanda.logger.info(f"npm - stdout: {output.stdout}")
-            self._redpanda.logger.info(f"npm - stderr: {output.stderr}")
+            retries = 2
+            while True:
```

Comment:
While retries > 0 ?

Reply:
It sorta looks nicer that way, but would risk falling out of the loop (and not raising an exception) if we ran out of retries and for some reason hadn't handled applied the break/continue/raise logic inside the loop.